iron-doc-property.css

```diff
@@ -23,6 +23,11 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   display: none;
 }
 
+.deeplink {
+  color: currentcolor;
+  text-decoration: none;
+}
+
```

Comment:
Maybe we could also add a `:hover` rule,  e.g. `text-decoration:underline` or a different font color.  wdyt?